### PR TITLE
Try clamav 0.99 in Travis

### LINKED ISSUE #143: Make CI work
As trusty's clamav version is less than 0.99:
Please add https://launchpad.net/~teward/+archive/ubuntu/clamav

`LC_ALL=en_US.UTF-8 sudo add-apt-repository ppa:teward/clamav -y`
https://docs.travis-ci.com/user/installing-dependencies/

### PR DESCRIPTION
**Pull request by szepeviktor**
Fixes #143

### CODE REVIEW

**Comment by szepeviktor** [comment]
:cry: "DOES NOT WORK FOR TRUSTY! It has a SystemD requirement!"

**Comment by szepeviktor** [comment]
BTW. It is very easy to backport claamv 0.99 from the wheezy source, but the easiest to just download 6 packages.

File: .travis.yml [hunk 1]
```diff
@@ -1,41 +1,24 @@
-#language: shell
+language: bash
 
-# Required environment variables from repository settings
```
Comment (szepeviktor) [inline .travis.yml]: I've opened a support ticket securiteinfo.com.
I hope they will give a Pro license for running Travis.

**Comment by szepeviktor** [comment]
All lights are green. With my personal `securiteinfo_authorisation_signature` it works OK if you comment out that two rules from #145

**Comment by szepeviktor** [comment]
This is a very nice project.

File: .travis.yml (hunk 1 above)
Comment (szepeviktor) [inline .travis.yml]: Answer from SecuriteInfo.com:

Hello Viktor,

> We are developing this updater for clamav:
> https://github.com/extremeshok/clamav-unofficial-sigs

Would you be so kind and give us a Pro licence for free?
I think the correct address is: admin@extremeshok.com
Could admin@extremeshok.com register an account @securiteinfo.com ?

> We will use it only in TravisCI builds.

The ID of pro account should never be published. Ok ?
BTW, what is TravisCI ?

-- 
Cordialement,

Arnaud Jacques
Gérant de SecuriteInfo.com

Comment (extremeshok) [inline .travis.yml]: I have an account with them under admin@extremeshok.com

Comment (szepeviktor) [inline .travis.yml]: OK!!
Could you start up Travis with an encrypted variable?

**Pull request closed without merging**